sap_hana/assets/dashboards/overview.json

```diff
@@ -0,0 +1,586 @@
+                        "conditional_formats": [],
+                        "q": "avg:sap_hana.backup.latest{*}/60",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": null
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "time": {
+                "live_span": "15m"
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Time since last backup",
+            "type": "query_value",
+            "width": 18,
+            "x": 1,
+            "y": 30
+        },
+        {
+            "add_timeframe": true,
+            "error": null,
+            "height": 15,
+            "isShared": false,
+            "legend": false,
+            "legend_size": "0",
+            "scaleFactor": 1,
+            "sharedGlobalTime": {
+                "live_span": "1h"
+            },
+            "tile_def": {
+                "autoscale": true,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "conditional_formats": [],
+                        "q": "avg:sap_hana.volume.io.throughput{*}",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "line"
+                    }
+                ],
+                "viz": "timeseries"
+            },
+            "time": {},
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Average throughput",
+            "type": "timeseries",
+            "width": 47,
+            "x": 69,
+            "y": 25
+        },
+        {
+            "add_timeframe": true,
+            "error": null,
+            "height": 15,
+            "isShared": false,
+            "legend": false,
+            "legend_size": "0",
+            "scaleFactor": 1,
+            "sharedGlobalTime": {
+                "live_span": "1h"
+            },
+            "tile_def": {
+                "autoscale": true,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "conditional_formats": [],
+                        "q": "avg:sap_hana.volume.io.utilized{*}, avg:sap_hana.volume.io.read.utilized{*}, avg:sap_hana.volume.io.write.utilized{*}",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "area"
```

Comment:
This chart seems not correct. The `area` is cumulative, it doesn't really make sense to add up percents with different meanings.